pages/index.tsx

```diff
@@ -67,9 +67,16 @@ export default function Home({ openProposals, fallback }: HomePageProps) {
         </div>
         <div className="md:w-2/3 md:ml-auto">
           <h1 className="text-3xl font-semibold  m-4 px-2 pt-2">
-            {selectedProposal
-              ? `${selectedProposal.id}: ${selectedProposal.title}`
-              : 'Vote Timeline'}
+            {selectedProposal ? (
+              <a
+                href={`https://nouns.wtf/vote/${selectedProposal.id}`}
+                target="_blank"
```

Comment:
add `rel="noreferrer"` and probably className hover:underline, also can use `getNounsLink` helper func 

be sure to run yarn lint:fix locally before pushing and fixing any errors (warnings OK)

Reply:
gotcha! Lemme fix that for ya.

Reply:
looks like we missed using [getNounsLink](https://github.com/wilsoncusack/nouns-vote-with-reason/blob/d0ff589ffcaab1884c6ccaec7d4cc754024baa35/lib/util/link.ts#L5), but it works :)

Reply:
ah sorry, I missed that.

Reply:
added!